### ISSUE TITLE
executeApplyOnlyIf { true } is not respected 

### ISSUE THREAD

**Issue by wuub** [JetBrains/jetbrains-gradle-plugins#1]
in `examples/project-b` I changed the whole tf file to just

```
resource "random_integer" "priority" {
  min = 1
  max = 50001
}
```

and then tried `gradle terraformApply` 

it always ends in 

```
> Task :examples:terraform:project-b:terraformMainApply SKIPPED
Please specify a criteria with which execute terraform apply:
terraform {
    executeApplyOnlyIf { System.getenv("CAN_EXECUTE_TF_APPLY") == "true" }
}
Or disable the check:
terraform {
    executeApplyOnlyIf { true }
}
Cannot execute terraformMainApply. Please check your terraform extension in the script.
```

is it some sort of race condition? does this happen https://github.com/JetBrains/jetbrains-gradle-plugins/blob/master/plugins/src/main/kotlin/org/jetbrains/gradle/plugins/terraform/PluginUtils.kt#L334 before extension is fully initialised? 

I don't know :)

**Comment by lamba92** [comment]
It might be due to the `main` source set being initialized too early. I'll fix it in the next release

**Comment by lamba92** [comment]
Should be fixed in `1.1.7`
